robot/src/main/java/org/frcteam2910/common/robot/drivers/Limelight.java

```diff
@@ -128,6 +200,9 @@ public void setStreamMode(StreamMode mode) {
         }
     }
 
+    /*
+     * Returns the NetworkTable being used to get values from the Limelight
```

Comment:
Start the sentence with `Gets` instead of `Returns`

Reply:
done